.vscode/tasks.json

```diff
@@ -9,7 +9,18 @@
 			],
 			"group": {
 				"kind": "build",
-				 "isDefault": true
+				"isDefault": true
+			}
+		},
+		{
+			"type": "npm",
+			"script": "watch",
+			"group": "build",
+			"problemMatcher": [],
+			"label": "npm: watch",
+			"detail": "npx tsc -w & nodemon -q -w dist",
+      		"runOptions": {
+				"runOn": "folderOpen"
```

Comment:
Nice improvement for developers!

Could you also jot a note in the README https://github.com/mseag/sfm-utils#compiling-sfm-utils ?